### ISSUE TITLE
Lints are failing

### ISSUE THREAD

**Issue by probably-neb** [WesternConcrete/307FinalProject#7]
We need to find a good time to fix all of the eslint, tsc, and prettier checks when it wont cause a merge conflict mess with all of the open branches and pull requests

**Comment by probably-neb** [comment]
Once this is done we can consider making it so you can't merge into main if the checks aren't passing
